utils/streams.js

```diff
@@ -0,0 +1,127 @@
+import commander from 'commander';
+import through from 'through2';
+import fs from 'fs';
+import path from 'path';
+import { promisify } from 'util';
+import csvjson from 'csvjson';
+const readFileAsync = promisify(fs.readFile);
+const readDirAsync = promisify(fs.readdir);
+
+function streams() {
+    commander.option('-a, --action <string>', 'action name taken')
+        .option('-f, --file <string>', 'file will read')
+        .option('-p, --path <string>', 'path for css folder')
+        .parse(process.argv);
+    executeTheAction();
+}
+const actionList = {reverse, transform, outputFile, convertFromFile, convertToFile};
+function executeTheAction() {
+    if (process.argv.length < 3) {
+        commander.help();
+    } else if (process.argv[2] == '-h' || process.argv[2] == '--help') {
+        commander.help();       
+    } else {
+        if (!!commander.action) {
+            if(actionList[commander.action] != undefined) {
+                actionList[commander.action]();
```

Comment:
Don't you need to pass some arguments in case `outputFile`, `convertFromFile`, etc.?

`actionList[commander.action](commander.file)`

meanwhile in case of `reverse` and `transform`, you don't pass any arguments:

`actionList[commander.action]()`

you can define variable 'argument' and use binary operator:

`const argument = (commander.action !== 'reverse' || commander.action !== 'transform') && (commander.file || commander.path)`
then execute appropriate function:
`actionList[commander.action](argument)`

Reply:
Thanks for the suggestion. I have updated the code.
I felt this would fail when I pass the args wrong.
Example: `node streams --action=outputFile --path=./css`
Instead of `file` if I pass `path` we should handle it as an exception so I have implemented reading the filename at the function level.